src/Ting/Repository/Hydrator.php

```diff
@@ -440,13 +445,12 @@ function (Metadata $metadata) use ($column, &$result) {
      */
     private function manageIfYouCan($entity)
     {
-        if (isset($entity->tingUUID) === true && isset($this->alreadyManaged[$entity->tingUUID]) === true) {
-            return;
+        if ($this->alreadyManaged === null) {
+            $this->alreadyManaged = new WeakMap();
```

Comment:
it cannot be null at this point

Reply:
Yes, you're right ! I have removed this code.